### DISCUSSION TITLE
Is the UI layout supposed to save state?

### DISCUSSION THREAD

**Discussion by tdtrumble** [Comfy-Org/ComfyUI_frontend#1582] [Q&A]
Every time I launch comfy the UI layout resets. I close the output stream thing at the bottom and drag the queue button into the top menu, then next time I launch I do it again. I'm using the portable windows version. Am I missing something? A setting to save the layout?

I also show/hide/drag a few boxes from custom nodes around each time, but I understand those wouldn't be under comfys control.

**Comment by LukeG89** [comment]
The position of Queue menu is stored in the browser, so everytime you clean browser cache, the menu is restored in the default position

**Reply by LukeG89** [reply]
@tdtrumble If you want, you can open a feature request about storing Queue position in local storage
